typescript/sdk/src/deploy/HyperlaneDeployer.ts

```diff
@@ -461,4 +461,29 @@ export abstract class HyperlaneDeployer<
     }
     return ret;
   }
+
+  protected async transferOwnershipOfContracts(
+    chain: ChainName,
+    owner: types.Address,
+    ownables: Ownable[],
+  ): Promise<ethers.ContractReceipt[]> {
+    const receipts: ethers.ContractReceipt[] = [];
+    for (const ownable of ownables) {
+      const currentOwner = await ownable.owner();
+      if (!utils.eqAddress(currentOwner, owner)) {
+        const receipt = await this.runIfOwner(chain, ownable, () =>
+          this.multiProvider.handleTx(
+            chain,
+            ownable.transferOwnership(
+              owner,
+              this.multiProvider.getTransactionOverrides(chain),
```

Comment:
I remember I considered ways to avoid this double MP call (one to handle and one to get tx overrides) when refactoring the MP but I didn't want the handleTx method to assume it should use the overrides. Maybe we should revisit that to simplify cases like these.

Reply:
Agreed but out of scope, IMO, for now